src/pages/components/dots.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Dots
+description: >
+  The Dots loader communicates ongoing activity after a user takes an action. It
+  tells them that something is taking place.
+package: '@zendeskgarden/react-loaders'
+components:
+  - loaders/src/elements/Dots.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To communicate a loading status after a user takes an action</li>
```

Comment:
Let's make this a little clearer:

To indicate that a single component, like a Button or Search input, is doing something